requirements-common.txt

```diff
@@ -20,7 +20,7 @@ tiktoken >= 0.6.0  # Required for DBRX tokenizer
 lm-format-enforcer >= 0.10.9, < 0.11
 outlines == 0.1.11
 lark == 1.2.2 
-xgrammar == 0.1.11; platform_machine == "x86_64"
+xgrammar >= 0.1.13; platform_machine == "x86_64"
```

Comment:
I don't think this is relevant anymore?

https://github.com/vllm-project/vllm/pull/14590